lib/membership/member.js

```diff
@@ -91,10 +90,14 @@ Member.prototype.evaluateUpdate = function evaluateUpdate(update) {
     if (this._isLocalOverride(update)) {
         // Override intended update. Assert aliveness!
         this.ringpop.stat('increment', 'refuted-update');
-        update = _.defaults({
+        var newIncNumber = this.Date.now();
+        update = {
+            source: this.ringpop.whoami(),
```

Comment:
I see, however, in `member.js` we have no access to the `localMember` which is accessible in `update.js`. In the `_isLocalOverride` function that tests if the update is about this node the `whoami` function is used as well.